sdk/storage/azblob/blockblob/client_test.go

```diff
@@ -587,6 +587,662 @@ func (s *BlockBlobRecordedTestsSuite) TestUploadBlockWithImmutabilityPolicy() {
 	_require.Nil(err)
 }
 
+func setUpPutBlobFromURLTest(s *BlockBlobUnrecordedTestsSuite, testName string, _require *require.Assertions, svcClient *service.Client) (*container.Client, *blockblob.Client, *blockblob.Client, string, []byte) {
+	containerName := testcommon.GenerateContainerName(testName)
+	containerClient := testcommon.CreateNewContainer(context.Background(), _require, containerName, svcClient)
+
+	srcBlob := testcommon.GenerateBlobName("src" + testName)
+	srcBBClient := testcommon.CreateNewBlockBlob(context.Background(), _require, srcBlob, containerClient)
+
+	dest := testcommon.GenerateBlobName("dest" + testName)
+	destBBClient := testcommon.CreateNewBlockBlob(context.Background(), _require, dest, containerClient)
+
+	// Upload some data to source
+	contentSize := 4 * 1024 // 4KB
+	r, sourceData := testcommon.GetRandomDataAndReader(contentSize)
+	_, err := srcBBClient.Upload(context.Background(), streaming.NopCloser(r), nil)
+	_require.Nil(err)
+
+	// Create SAS for source and get SAS URL
+	expiryTime, err := time.Parse(time.UnixDate, "Fri Jun 11 20:00:00 UTC 2049")
+	_require.Nil(err)
+
+	credential, err := testcommon.GetGenericSharedKeyCredential(testcommon.TestAccountDefault)
+	if err != nil {
```

Comment:
This can be used instead.
```suggestion
	_require.NoError(err)
```